eos-tech-support/eos-diagnostics

```diff
@@ -673,6 +658,26 @@ function dumpHardwareInfo(filename) {
     return true;
 }
 
+function dumpHardwareInfo(filename) {
+    return dumpToArchive(filename, function(dir) {
+        copyAcpiFirmware(dir);
+
```

Comment:
Looks like the `copyCoredumps(dir);` line was omitted here.  Was that intentional?  If so, it seems that he commit message is incorrect that there are no functional changes.